pkg/model/stack_serializer_test.go

```diff
@@ -1588,6 +1601,13 @@ func Test_Environment(t *testing.T) {
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
 
+			if tt.setEnvar {
```

Comment:
We could always set this variable. That way we can remove `tt.setEnvar` and the code is cleaner